### PR TITLE
Make graphs full width

### PR DESCRIPTION
**Pull request by moritzpflueger**
### findings
it seems **echarts.js** and `.collapsible-body` class do not work good together.

### done:
* i disabled collapsibility (deleted class) for now, until i can further investigate the problem. I'll make a new issue for that (see #94 )
* charts are now full width, because i changed the container width to 100% and made use of the `resize()` from echartjs. (This only works with collabsible disabled)
* had to scope some vars, because the resizing function seems to only work on the last chart-object. With every chart having its own var name, it's now working correctly

edit: add issue reference

### CODE REVIEW

**Comment by moritzpflueger** [comment]
Hey, of course I can also keep this PR open until the open issue #94 described above is solved. And include it in this PR. What do you think @pinpox ?

**Comment by moritzpflueger** [comment]
Okay, I'll continue pushing into this PR. Also I think tabs are a great idea!

**Comment by moritzpflueger** [comment]
ready for review

**Comment by pinpox** [comment]
Nice, thanks for the work! I'll pull and review, excited to get this one fixed.

**Comment by pinpox** [comment]
This looks great! 

Just a  very minor issue, 

image

The two graphs are misaligned, Is it possibe to make them both centered?

File: templates/details.html [hunk 1]
```diff
@@ -36,52 +36,57 @@ <h3>{{.PlayerSummary.Personaname}}</h3>
...
-		<div class="collapsible-body white">
-			{{template "chart-overview.html" .}}
-			{{template "chart-accuracy.html" .}}
-			{{template "chart-adr.html" .}}
-			{{template "table-other.html" .}}
-		</div>
-	</li>
-	<li>
-		<div class="collapsible-header blue-grey darken-2 white-text"><i class="material-icons">multiline_chart</i>Last Match</div>
-		<div class="collapsible-body white">
```
Comment (pinpox) [inline templates/details.html:51]: #93

File: templates/details.html [hunk 2]
```diff
@@ -36,52 +36,57 @@ <h3>{{.PlayerSummary.Personaname}}</h3>
...
-<script charset="utf-8">
-	document.addEventListener('DOMContentLoaded', function() {
-	var elems = document.querySelectorAll('.collapsible');
-	var instances = M.Collapsible.init(elems, options);
-  });
-
-  // Or with jQuery
+<div class="row blue-grey">
+	<div class="col s12">
+		<ul class="tabs blue-grey">
```
Comment (pinpox) [inline templates/details.html:41]: Maybe add the `white` class somewhere here back in.

**Comment by pinpox** [comment]
I will merge the request now, since it already works nicely. We can look at the colors separtely afterwards. Thanks for the work!

**Pull request merged**